docs/index.mdx

```diff
@@ -7,6 +7,22 @@ A smart contract is a program that verifies and executes the performance of a co
 Programs that run on blockchains are commonly referred to as smart contracts because they mediate important functionality (such as currency)
 without having to rely on a central authority (like a bank).
 
+## Why should you use Cadence for building smart contracts ?
+
+### 1) **Security and Safety**
+
+Cadence provides security and safety guarantees that greatly simplify the development of secure smart contracts. As smart contracts often deal with valuable assets, Cadence provides the resource-oriented programming paradigm, which guarantees that assets can only exist in one location at a time, cannot be copied, and cannot be accidentally lost or deleted. Cadence includes several language features which prevent entire classes of bugs.
+These security and safety features allow smart contract developers to focus on the business logic of their contract instead of preventing accidents and attacks.
+
+### 2) Composability
+
+Cadence enables composability. Resources (which are arbitrary user-defined data types) are stored directly in users’ accounts, and can flow freely between contracts: They can be passed as arguments to functions, returned from functions, or even combined in arbitrary data structures. This makes implementing business logic easier, more natural and promotes reuse of existing logic.
+
+### 3) **Simplicity**
+
+Cadence’s syntax is inspired by popular modern general-purpose programming languages like **[Swift](https://developer.apple.com/swift/)**, **[Kotlin](https://kotlinlang.org/)**, and **[Rust](https://www.rust-lang.org/),** so developers will find the syntax and the semantics familiar. 
+Practical tooling, documentation, and examples enable developers to start creating programs quickly and effectively. Hundreds of developers were able to learn Cadence quickly and develop production-quality smart contracts with it shortly.
+
```

Comment:
Adding line breaks makes it easier to edit/review this in the future:

```suggestion
## Why should you use Cadence for building smart contracts ?

### 1) **Security and Safety**

Cadence provides security and safety guarantees that greatly simplify the development of secure smart contracts. 
As smart contracts often deal with valuable assets, Cadence provides the resource-oriented programming paradigm, which guarantees that assets can only exist in one location at a time, cannot be copied, and cannot be accidentally lost or deleted. Cadence includes several language features which prevent entire classes of bugs.
These security and safety features allow smart contract developers to focus on the business logic of their contract instead of preventing accidents and attacks.

### 2) Composability

Cadence enables composability. 
Resources (which are arbitrary user-defined data types) are stored directly in users’ accounts, and can flow freely between contracts: 
They can be passed as arguments to functions, returned from functions, or even combined in arbitrary data structures. This makes implementing business logic easier, more natural and promotes reuse of existing logic.

### 3) **Simplicity**

Cadence’s syntax is inspired by popular modern general-purpose programming languages like **[Swift](https://developer.apple.com/swift/)**, **[Kotlin](https://kotlinlang.org/)**, and **[Rust](https://www.rust-lang.org/),** so developers will find the syntax and the semantics familiar. 
Practical tooling, documentation, and examples enable developers to start creating programs quickly and effectively. 
Hundreds of developers were able to learn Cadence quickly and develop production-quality smart contracts with it shortly.

```